userspace/falco/configuration.cpp

```diff
@@ -212,6 +212,10 @@ void falco_configuration::init(string conf_filename, const vector<string> &cmdli
 	m_webserver_ssl_enabled = m_config->get_scalar<bool>("webserver.ssl_enabled", false);
 	m_webserver_ssl_certificate = m_config->get_scalar<string>("webserver.ssl_certificate", "/etc/falco/falco.pem");
 
+	// we put this value in the configuration file because in this way we can change the dimension
+	// at every reload.
+	m_single_buffer_dimension = m_config->get_scalar<uint64_t>("single_buffer_dimension", 0);
```

Comment:
Should or could we be doing more to ensure end user parameter input is not bogus? This would apply to any parameter and is not specific to this PR.

Reply:
For what concern the validity checks for sure we have to implement them in the final drivers since they are the end users of this info, we can put also something in Falco, BTW if we use the page number as a value there is not too much to check probably that it is greater than `0` and that it is a power of `2`, right?